src/Microsoft.AspNet.Mvc.Razor/IInitializeTagHelperOfT.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace Microsoft.AspNet.Mvc.Razor
+{
+    /// <summary>
+    /// Initializes an <see cref="ITagHelper"/> before it's executed.
+    /// </summary>
+    /// <typeparam name="TTagHelper">The <see cref="ITagHelper"/> type.</typeparam>
+    public interface IInitializeTagHelper<TTagHelper>
```

Comment:
other repos may have other conventions.  and we have a few oddballs like `IConsumesActionConstraint`, `IAllowAnonymous`, and of course `ICanHasViewContext`.  but  `ITagHelperInitializer` is much more consistent with most other interfaces we define in MVC.